src/ray/gcs/gcs_server/gcs_actor_manager.h

```diff
@@ -149,17 +147,20 @@ class GcsActor {
   rpc::ActorTableData *GetMutableActorTableData();
   rpc::TaskSpec *GetMutableTaskSpec();
 
-  std::shared_ptr<const GcsActorWorkerAssignment> GetActorWorkerAssignment() const;
-
-  void SetActorWorkerAssignment(std::shared_ptr<GcsActorWorkerAssignment> assignment_ptr);
+  const ResourceRequest &GetAcquiredResources() const;
+  ResourceRequest *GetMutableAcquiredResources();
```

Comment:
Use `void SetAcquiredResources(ResourceRequest &&resource_request);` to instead of `ResourceRequest *GetMutableAcquiredResources();`